docs/api_abi-kybernetwork.md

```diff
@@ -1050,10 +1050,11 @@ These amounts should be in the source and destination token decimals respectivel
 #### `maxDestAmount`
 
 This parameter should never be zero. Set to an arbitarily large amount for all source tokens to be converted.
+When using `maxDestAmount`, we recommend using `slippageRate` or add a buffer to `expectedRate` rather than using the exact rate from getExpectedRate. Otherwise, it is likely that the trade will revert.
```

Comment:
if we only ask to add buffer.
its compatible with Katalyst.
can consider that.

Also we have a comment from Spyros that calling this slippage rate is wrong terminology. worstRate is better.

and best is let the user decide and consider his risks.
notice we decided to remove this value for get expected rate API.